### ISSUE TITLE
my  loss looks extremely unusual with  my own data, 

### ISSUE THREAD

**Issue by nuass** [divamgupta/deep_clustering_kingdra#7]
my dataset categories number is 631, the program logs display that loss  is  even smaller than 20news   ,Is my setting improper?
 I set  config file like this :
{ 

    "training_path": "ens_cluster_datasets/xx/train_X.npy" ,
    "ground_truth_path" : "ens_cluster_datasets/xx/train_Y.npy"  ,
    "n_epochs_per_iter":5 ,
    "n_epochs_per_iter0":5 ,
    "batch_size":2048 ,
    "n_iter":10  ,
    "noise_std" : 0.2 ,
    "mi_loss_w_init":0.3 ,
    "ladder_loss_w_init":1 ,
    "noise_kl_w_init":1 ,
    "last_layer_bn":false ,
    "n_class" : 631 ,
    "fc_layer_sizes":[  1200  , 1200 , 631 ] ,
    "thresh":0

}

the program log  :

Epoch 1/5
2021-02-26 09:48:40.609729: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcublas.so.10.0
 - 10s - loss: -7.1115e-01
Epoch 2/5
 - 6s - loss: -2.6099e-01
Epoch 3/5
 - 7s - loss: 0.0490
Epoch 4/5
 - 7s - loss: 0.2618
Epoch 5/5
 - 6s - loss: 0.4247
/root/anaconda3/envs/zcctf/lib/python3.6/site-packages/keras/engine/training.py:297: UserWarning: Discrepancy between trainable weights and collected trainable weights, did you set `model.trainable` without calling `model.compile` after ?
  'Discrepancy between trainable weights and collected trainable'
Epoch 1/5
 - 9s - loss: -7.6037e-01
Epoch 2/5
 - 7s - loss: -1.7202e-01
Epoch 3/5
 - 6s - loss: 0.1432
Epoch 4/5
 - 6s - loss: 0.3263
Epoch 5/5
 - 6s - loss: 0.4878
/root/anaconda3/envs/zcctf/lib/python3.6/site-packages/keras/engine/training.py:297: UserWarning: Discrepancy between trainable weights and collected trainable weights, did you set `model.trainable` without calling `model.compile` after ?
  'Discrepancy between trainable weights and collected trainable'
Epoch 1/5
 - 10s - loss: -7.5107e-01
Epoch 2/5
 - 7s - loss: -1.5359e-01
Epoch 3/5
 - 7s - loss: 0.1047
Epoch 4/5
 - 6s - loss: 0.3391
Epoch 5/5
 - 6s - loss: 0.4869
